services/concept_insights/ci.js

```diff
@@ -175,13 +173,13 @@ module.exports = function (RED) {
       setupConceptInsightsNode(msg,config,this,function(concept_insights,account_id) {
         
         //Set up parameters, searching the latest wikipedia graph
-        if (!config.query) {
+        if (!msg.payload || typeof(msg.payload) != 'string') {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2394968359/issues/source?bid=3225115&fileBranchId=3230627#l177)